delayed_job/spec/monitor_delayed_jobs_spec.rb

```diff
@@ -19,6 +19,10 @@ def use_transactional_fixtures
   after(:each) do
     MonitorDelayedJobs::DelayedJob.delete_all
   end
+
+  it "has options that are valid YAML" do
+    expect{YAML.load(described_class::OPTIONS)}.not_to raise_error
```

Comment:
I dislike `not_to raise_error` as an expectation. Specs should tell the reader what *is* the expected outcome.